INSTALL.md

```diff
@@ -0,0 +1,315 @@
+# Installation
+
+## Requirements
+* [Java 1.8](http://www.oracle.com/technetwork/java/javase/downloads/index.html) or higher.
+* [Java Cryptography Extension](http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html) to
+  use Client-side Encryption with stronger ciphers (192/256-bit).
+* [Maven 3.3.x](https://maven.apache.org/) to contribute to the project.
+
+### CLI Requirements
```

Comment:
I"m not sure what is "CLI" specific here.  "CSE" intended?

Reply:
CLI is referring to the Java module java-manta-cli. When you are running it stand-alone it has different runtime requirements.